lib/Optimizer/PassManager.cpp

```diff
@@ -0,0 +1,50 @@
+#include "PassManager.hpp"
+
+#include <llvm/Support/WithColor.h>
+
+namespace glu::optimizer {
+
+PassManager::PassManager(
+    DiagnosticManager &diagManager, SourceManager &sourceManager,
+    llvm::raw_ostream &output, gil::Module *module,
+    llvm::BumpPtrAllocator &GILFunctionsArena
+)
+    : _diagManager(diagManager)
+    , _sourceManager(sourceManager)
+    , _output(output)
+    , _printer(&sourceManager, output)
+    , _module(module)
+    , _GILFunctionsArena(GILFunctionsArena)
```

Comment:
var names shouldn't start with capital, just `arena` and `_arena` is fine (or take a gilgen::GlobalContext?)